Solutions/SAP BTP/Analytic Rules/BTP - Mass user deletion in a sub account.yaml

```diff
@@ -0,0 +1,49 @@
+id: 6f1e58bd-cd95-4dfb-8883-94207f30929a
+kind: Scheduled
+name: BTP - Mass user deletion in a sub account
+description: Identifies user account deletion activity where the amount of deleted
+  users exceeds a predefined threshold.
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: SAPBTPAuditLog
+    dataTypes:
+      - SAPBTPAuditLog_CL
+queryFrequency: 15m
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1531
+  - T1485
+  - T1489
+  - T0813
+  - T0826
+  - T0827
+query: |
+  let bulk_delete_threshold = 10;
+  SAPBTPAuditLog_CL
+  | where isnotnull(Message.object)
+  | where Message.object has "scim user" and Message.object has "DELETE"
+  | mv-expand Attributes = Message.attributes
+  | mv-expand Attributes
```

Comment:
Do we require to mv-expand twice? Can you please provide sample logs for this data type?

Reply:
Removed superfluous mv-expand. Data samples can be found [here](https://github.com/kingwil/Azure-Sentinel/blob/wiking-sapbtp/Sample%20Data/SAPBTPAuditLog_RawLogs.json)